packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -0,0 +1,274 @@
+            _product, scoreAttributeIds);
+
+    List<String> importantAttributeIds =
+        productPreferences.getOrderedImportantAttributeIds();
+    importantAttributeIds = importantAttributeIds
+        .where((String attributeId) => !scoreAttributeIds.contains(attributeId))
+        .toList();
+    final List<Attribute> importantAttributes =
+        AttributeListExpandable.getPopulatedAttributes(
+            _product, importantAttributeIds);
+    final Widget attributesContainer = Container(
+        alignment: Alignment.topLeft,
+        margin: const EdgeInsets.fromLTRB(0, 16, 0, 16),
+        child: Column(children: <Widget>[
+          Wrap(runSpacing: 16, children: <Widget>[
+            for (final Attribute attribute in importantAttributes)
+              getAttributeChip(attribute, screenSize) ?? Container(),
+          ])
+        ]));
+
+    final List<Widget> listItems = <Widget>[];
+    listItems.add(Align(
+        heightFactor: 0.7,
+        alignment: Alignment.topLeft,
+        child: _buildProductImagesCarousel(context)));
+    listItems.add(
+      SmoothCard(
+        padding: const EdgeInsets.only(
+          right: 8.0,
+          left: 8.0,
+          top: 4.0,
+          bottom: 20.0,
+        ),
+        insets: const EdgeInsets.all(12.0),
+        child: Column(children: <Widget>[
+          Align(
+              alignment: Alignment.topLeft,
+              child: ListTile(
+                contentPadding: EdgeInsets.zero,
+                title: Text(
+                  _getProductName(appLocalizations),
+                  style: themeData.textTheme.headline4,
+                ),
+                subtitle:
+                    Text(_product.brands ?? appLocalizations.unknownBrand),
+                trailing: Text(
+                  _product.quantity ?? '',
+                  style: themeData.textTheme.headline3,
+                ),
+              )),
+          for (final Attribute attribute in scoreAttributes)
+            ScoreAttributeCard(attribute: attribute, iconHeight: iconHeight),
+          attributesContainer
+        ]),
+      ),
+    );
+    return ListView(children: listItems);
+  }
+
+  Widget? getAttributeChip(Attribute attribute, Size screenSize) {
+    final String? attributeDisplayTitle = getDisplayTitle(attribute);
+    final Widget attributeIcon = getAttributeDisplayIcon(attribute);
+    if (attributeDisplayTitle == null) {
+      return null;
+    }
+    return LayoutBuilder(
+        builder: (BuildContext context, BoxConstraints constraints) {
+      return SizedBox(
+          width: constraints.maxWidth / 2,
+          child: Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                attributeIcon,
+                Expanded(
+                    child: Text(
+                  attributeDisplayTitle,
+                ))
+              ]));
```

Comment:
Use trailing commas

Reply:
would be nice if Flutter format did this for us